lib/wpcom-error-handler/wpcom-error-handler.php

```diff
@@ -155,29 +171,32 @@ function wpcom_custom_error_handler( $whether_i_may_die, $type, $message, $file,
 		}
 
 		if ( 'stderr' === $display_errors && defined( 'STDERR' ) && is_resource( STDERR ) ) {
-			fwrite( STDERR, sprintf( $display_errors_format, $string, $message, $file, $line, htmlspecialchars( $source ), htmlspecialchars( $backtrace ) ) );
+			fprintf( STDERR, $display_errors_format, $string, htmlspecialchars( $message ), htmlspecialchars( $file ), htmlspecialchars( $line ), htmlspecialchars( $source ), htmlspecialchars( $backtrace ) );
```

Comment:
Since this PR focuses on formatting and coding standards changes, we should leave out the additional wrapping of `$message`, `$file` or `$line` with `htmlspecialchars()`.

Reply:
Fixed in 13425434c.

I hope I will not forget to create a PR fixing this :-)